SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -998,9 +998,10 @@ void FSpatialGDKEditorToolbarModule::CloudDeploymentClicked()
 	SpatialGDKEditorSettings->bUseDevelopmentAuthenticationFlow = true;
 	USpatialGDKSettings* SpatialGDKSettings = GetMutableDefault<USpatialGDKSettings>();
 
-	FString DevAuthToken;
-	if (!SpatialCommandUtils::GenerateDevAuthToken(SpatialGDKSettings->IsRunningInChina(), DevAuthToken))
+	FString DevAuthToken,ErrorMessage;
```

Comment:
I think we generally have each definition on a separate line, so:
```
FString DevAuthToken;
FString ErrorMessage;
```

@improbable-valentyn  to double-check, cause I'm actually not sure if this is part of our coding standard or whether this is also good. I just can't remember seeing it anywhere else in our codebase.